CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## [Unreleased]
 
+## [0.29.0] - 2018-12-18
+
+### Changed
+*  the core sends simultaneously to the INBOX now; for these purposes, users shoud create a 3rd thread calling the new functions dc_perform_mvbox_fetch() and dc_perform_mvbox_idle()
```

Comment:
We should probably call these methods as well in `src/module.c`

Reply:
jupp :)

@ralphtheninja i think it's very similar to https://github.com/deltachat/deltachat-node/blob/master/src/module.c#L159 - without the perform_jobs(), sth. like 

```
static void mvbox_thread_func(void* arg)
{
  dcn_context_t* dcn_context = (dcn_context_t*)arg;
  dc_context_t* dc_context = dcn_context->dc_context;

#ifdef NODE_10_6
  napi_acquire_threadsafe_function(dcn_context->threadsafe_event_handler);
#endif

  while (dcn_context->loop_thread) {
    dc_perform_mvbox_fetch(dc_context);
    dc_perform_mvbox_idle(dc_context);
  }

#ifdef NODE_10_6
  napi_release_threadsafe_function(dcn_context->threadsafe_event_handler, napi_tsfn_release);
#endif
}
```

and further down

```
uv_thread_create(&dcn_context->mvbox_thread, mvbox_thread_func, dcn_context);
```